### ISSUE TITLE
Block Worldedit in Unowned Chunks

### ISSUE THREAD

**Issue by ianmcvann** [cjburkey01/ClaimChunk#150]
**Is your feature request related to a problem? Please describe.**
As I'm building a creative build server, I'd like to provide users with worldedit functionalities. Griefing is a very prevalent issue, and as it stands now I'm not able to do so with this current plugin, as anyone with worldedit can change blocks outside their chunks.

**Describe the solution you'd like**
I would like a flag or restriction added (I know certain plugins add this to worldguard) that makes it so people cannot use Worldedit or any of its alternatives (FAWE, AsyncWE) in a claim.

**Describe alternatives you've considered**
Plugins like GriefPrevention support this with addons, something like that here may work.

**Additional context**
N/A

**Comment by cjburkey01** [comment]
Currently, there is WorldGuard support, but nothing for WorldEdit yet. So, essentially, something that checks if the WorldEdit selection intersects any claimed chunks for which the user doesn't have permission to edit?

**Comment by ianmcvann** [comment]
Correct. Basically just stopping anyone from using worldedit if their
selection goes outside their claim chunks.

**Comment by cjburkey01** [comment]
That would be a pretty good option to put into the new system that handles this sort of thing, so I'll keep this in mind as I'm still working on it and nowish would be the time to add it.

**Comment by ianmcvann** [comment]
Perfect! Yeah, that's one feature we'd love to give to our players right now that we can't yet.
